maakay-bot.py

```diff
@@ -447,7 +447,7 @@ async def challenge_history(ctx, user:discord.Member):
     if Challenge.objects.filter(Q(challenger=obj) | Q(contender=obj)).exists():
 
         challenges = await sync_to_async(Challenge.objects.filter(Q(challenger=obj) | Q(contender=obj))).order_by('-created_at')[:5]
-        embed = discord.Embed(title=f"{obj.name}'s Challenge History")
+        embed = discord.Embed(title=f"{str(user)[0:5]}'s Challenge History")
```

Comment:
could we get over this by having the user in the embed? (if the end goal is to know whose history we're checking)
`embed.set_author(name=user.name, icon_url=user.avatar_url)`